searches/interpolation_search.py

```diff
@@ -10,6 +10,8 @@ def interpolation_search(sorted_collection, item):
     :param sorted_collection: some ascending sorted collection with comparable items
     :param item: item value to search
     :return: index of found item or None if item is not found
+    >>> interpolation_search([-1, 0, 1], -1)
+    0
```

Comment:
Please add some tests that return useful/interesting values instead of just `0` and `None`.
Please add some tests that raise an exception like floats, str, list, or dicts.
Also: (tuple and set instead of list)
`>>> interpolation_search((-1, 0, 1), -1)`
`>>> interpolation_search({-1, 0, 1}, -1)`
`>>> interpolation_search([], -1)`

Reply:
Added more tests based on your advice